src/commands/mod.rs

```diff
@@ -737,4 +1066,53 @@ settings:
 "#
         );
     }
+
+    #[test]
+    fn test_prepare_manifest_payload_includes_some_fields() {
```

Comment:
No. This approach is too low level. As I mentioned it should be tested in the test_upload. You should check what actual payload ends up being.

Reply:
Addressed with -> https://github.com/Screenly/cli/pull/96/commits/10ec4da1b1d7a81371fe780eba68dc2c2784ea82